src/accessories/contact-sensor.js

```diff
@@ -0,0 +1,35 @@
+import 'babel-polyfill';
+import {HomeWizardBaseAccessory} from './accessory';
+
+export class HomeWizardContactSensor extends HomeWizardBaseAccessory {
+
+  model = 'Contact sensor';
+
+  setupServices() {
+    // Setup services
+    const contactSensorService = new this.hap.Service.ContactSensor();
+    contactSensorService
+      .getCharacteristic(this.hap.Characteristic.ContactSensorState)
+      .on('get', this.getContactSensorState.bind(this));
+
+    this.services.push(contactSensorService);
+  }
+
+  getContactSensorState(callback) {
+    this.api.getStatus(this.id, 'kakusensors').then(sensor => {
+      const contact = sensor.status === null
+        ? this.hap.Characteristic.ContactSensorState.CONTACT_NOT_DETECTED
+        : this.hap.Characteristic.ContactSensorState.CONTACT_DETECTED;
+
+      if (contact === this.hap.Characteristic.ContactSensorState.CONTACT_DETECTED) {
+        this.log(`Detected contact at sensor:${this.name}`);
```

Comment:
Isn't it more useful to log when there is no contact? Think most of the times contact sensors are used in the why to do something when contact breaks? Not sure though, don't use them myself ;-)